graphics/src/MeshManager.cc

```diff
@@ -145,21 +151,30 @@ const Mesh *MeshManager::Load(const std::string &_filename)
     std::transform(extension.begin(), extension.end(),
         extension.begin(), ::tolower);
     MeshLoader *loader = nullptr;
-
-    loader = &this->dataPtr->assimpLoader;
-    /*
-    if (extension == "stl" || extension == "stlb" || extension == "stla")
-      loader = &this->dataPtr->stlLoader;
-    else if (extension == "dae")
-      loader = &this->dataPtr->colladaLoader;
-    else if (extension == "obj")
-      loader = &this->dataPtr->objLoader;
+    this->SetAssimpEnvs();
+    if (this->dataPtr->forceAssimp)
+    {
+      loader = &this->dataPtr->assimpLoader;
+    }
     else
     {
-      gzerr << "Unsupported mesh format for file[" << _filename << "]\n";
-      return nullptr;
+      if (extension == "stl" || extension == "stlb" || extension == "stla")
+        if (this->dataPtr->useAssimp)
```

Comment:
It looks like `useAssimp` is only used to enable its use for STLs. Can't we just rely on `forceAssimp` for that, and reduce the number of environment variables?

Reply:
Hi @chapulina,
Is there a way to document these variables in the MeshManager's header without moving them there?

Reply:
Is [this](https://github.com/gazebosim/gz-common/pull/403/commits/94bf4c04771205c1846bc5e6055023ca2ed4eeaa) commit what you taught?

Reply:
[Here](https://github.com/gazebosim/gz-common/pull/403/commits/27b8a2b154c9a3e014831391198eb38961cbc63d)

Reply:
Great, thanks!